polymetis/examples/README.md

```diff
@@ -0,0 +1,12 @@
+
+## Get started
+
+To run the code you need to do the following: 
```

Comment:
Some more overview here would be good -- what is accomplished when these commands are run?

Reply:
Good point, yes I will add more content here.